requirements-dev.txt

```diff
@@ -59,3 +59,4 @@ nvidia-ml-py
 huggingface_hub
 pyamg>=5.0.0
 git+https://github.com/Project-MONAI/GenerativeModels.git@7428fce193771e9564f29b91d29e523dd1b6b4cd
+packaging
```

Comment:
```suggestion
```
No need this package here, requirement-min already include it.
https://github.com/Project-MONAI/MONAI/blob/de41c6403f83d53bc85563a23912aff83e49d5f6/requirements-dev.txt#L2

Reply:
Hi @K-Rilla, could you please remove the packaging here, it's duplicated. Thanks!